db/migrate/20170630090000_add_category_foreign_key.rb

```diff
@@ -7,6 +7,6 @@
 
   down do
     execute "ALTER TABLE visualizations DROP CONSTRAINT visualizations_category_fkey;"
-    execute "ALTER TABLE visualizations DROP CONSTRAINT unique_id;"
+    execute "ALTER TABLE visualization_categories DROP CONSTRAINT unique_id;"
```

Comment:
Should migration scripts be modified?  Shouldn't we simply add a new migration script to make any changes?

Reply:
this one is a bug fix actually

Reply:
Correct, and it is also a change to the down task, not to the up task for an applied migration script, so it should be modified.